frontend/sw360-portlet/src/main/webapp/html/utils/includes/editAttachments.jspf

```diff
@@ -168,8 +168,9 @@
         // register event handlers
         $('#upload-attachment-button').on('click', function() {
             attachmentUploader.showUploadDialog(function(addedAttachment) {
-                attachmentTable.row.add(addedAttachment).draw(false);
-                $(attachmentTable.row('#' + addedAttachment.attachmentContentId).node()).addClass('added');
+            	attachmentTable.row.add(addedAttachment).draw(false);
+            	$(attachmentTable.row('#' + addedAttachment.attachmentContentId).node()).addClass('added');
+
```

Comment:
Maybe you could reset this file as nothing valuable got changed?

Reply:
Done